packages/picasso/src/utils/useBoolean/use-boolean.ts

```diff
@@ -0,0 +1,30 @@
+import { useState } from 'react'
+
+/** Set value to true */
+type SetTruthy = () => void
+/** Set value to false */
+type SetFalsy = () => void
+/** Toggle the value */
+type Toggle = () => void
+type IsTruthy = boolean
+
+type UseBooleanType = (
+  defaultValue?: boolean
+) => [IsTruthy, SetTruthy, SetFalsy, Toggle]
+
+/**
+ *
+ * @param defaultValue?: Boolean
+ * @returns [isTruthy, setTruthy(), setFalsy(), toggle()]
+ */
+const useBoolean: UseBooleanType = (defaultValue = false) => {
```

Comment:
Interesting hook :thinking: 

Technically:
```suggestion
 * @returns [isTruthy, setTruthy, setFalsy, toggle]
 */
const useBoolean: UseBooleanType = (defaultValue = false) => {
```

Reply:
done and moved to separate PR https://github.com/toptal/picasso/pull/3258